traffic_ops/traffic_ops_golang/deliveryservice/safe.go

```diff
@@ -23,11 +23,10 @@ import (
 	"database/sql"
 	"fmt"
 	"github.com/apache/trafficcontrol/lib/go-log"
-	"net/http"
-
 	"github.com/apache/trafficcontrol/lib/go-tc"
 	"github.com/apache/trafficcontrol/traffic_ops/traffic_ops_golang/api"
 	"github.com/apache/trafficcontrol/traffic_ops/traffic_ops_golang/tenant"
+	"net/http"
```

Comment:
We try to separate imports with newlines for "standard library," "third party", and "our own," just to make it a little easier to read. Would you mind moving this up with the other standard library imports? Mind also moving `lib/go-log` down with the other TC ones? (that one wasn't yours, but while you're improving it, might as well do that one too).